docker/release/Dockerfile

```
@@ -1,11 +1,15 @@
 FROM ubuntu:18.04
 
-RUN apt-get update
+RUN apt-get update; \
+    apt install -y software-properties-common; \
+    add-apt-repository ppa:ubuntu-toolchain-r/test
 
 #Install iroha
 COPY iroha.deb /tmp/iroha.deb
 RUN apt-get install -y /tmp/iroha.deb; \
```

```suggestion
RUN set -e; apt-get install -y /tmp/iroha.deb; \
```
we should fail if iroha is not instaled